components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -2414,10 +2550,10 @@ def definition_body(self):
         for arrayName in self.properties.arrayNames():
             array = getattr(self.properties, arrayName)
             if arrayName == "consts":
-                if array.length():
-                    properties[arrayName] = array.variableName()
+                if not array.length():
+                    properties[arrayName] = "EMPTY_CONSTANTS"
                 else:
-                    properties[arrayName] = "&[]"
+                    properties[arrayName] = array.variableName()
```

Comment:
Please revert this. It worked before and you didn't even document `EMPTY_CONSTANTS`.